Formula/s/swiftly.rb

```diff
@@ -0,0 +1,29 @@
+class Swiftly < Formula
+  desc "Swift toolchain installer and manager"
+  homepage "https://github.com/swiftlang/swiftly"
+  url "https://github.com/swiftlang/swiftly.git",
+      tag:      "1.0.0",
+      revision: "a9eecca341e6d5047c744a165bfe5bbf239987f5"
+  license "Apache-2.0"
+  head "https://github.com/swiftlang/swiftly.git", branch: "main"
+
+  depends_on macos: :ventura
+  depends_on xcode: "8.0"
+
+  uses_from_macos "swift" => :build, since: :sonoma # swift 5.10+
+  uses_from_macos "zlib"
+
+  def install
+    args = if OS.mac?
+      ["--disable-sandbox"]
+    else
+      ["--static-swift-stdlib"]
+    end
+    system "swift", "build", *args, "--configuration", "release", "--product", "swiftly"
```

Comment:
I think if you add `-Xcc -I#{HOMEBREW_PREFIX}/include -Xlinker -L#{HOMEBREW_PREFIX}/lib` then it will work

Related: https://github.com/Homebrew/homebrew-core/blob/master/Formula/u/unxip.rb#L33

Reply:
> I think if you add -Xcc -I#{HOMEBREW_PREFIX}/include -Xlinker -L#{HOMEBREW_PREFIX}/lib then it will fix the zlib problem.

You can even guard it under the if above to only do it on Linux (since it's only necessary for Linux)

Reply:
can we patch package.swift to include the change, looks like GzipSwift is no longer maintained.

Reply:
Sorry, not sure that I follow

Do you say that `-Xcc -I#{HOMEBREW_PREFIX}/include -Xlinker -L#{HOMEBREW_PREFIX}/lib` won't work?

I mean - if it does, it looks a clearer solution than patching `Package.swift`

Reply:
Looks like this doesn't work. I've built the formula locally where it worked, but probably only because it used the system headers and libraries. These flags don't seem to be passed to the underlying C compiler calls.

@chenrui333: Do you mean to patch a `.systemLibrary` target into the `Package.swift`?

Reply:
We went with a macOS-only formula for now. This is up to be resolved later by someone with more experience in this area.